src/ripple/app/ledger/LedgerMaster.cpp

```diff
@@ -1558,12 +1546,13 @@ LedgerMaster::~LedgerMaster ()
 {
 }
 
-bool LedgerMaster::shouldAcquire (std::uint32_t currentLedger,
```

Comment:
LedgerINdex is a typedef for a std:uint32_t  Each of the params signify a ledger index of type std_unit32_t.
I think that unless all of them were changed, the inconsistency could be more confusing rather than helpful.